packages/preview/codelst/2.0.1/typst.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "codelst"
+version = "2.0.1"
+entrypoint = "codelst.typ"
+authors = ["Jonas Neugebauer"]
+license = "MIT"
+description = "A typst package to render sourcecode."
+repository = "https://github.com/jneug/typst-codelst"
+exclude = ["example.typ", "example.pdf", "manual.pdf", "manual.typ", "tbump.toml"]
+compiler = "0.0.9"
+keywords = ["sourcecode", "code", "syntax-highlighting", "raw", "line numbers"]
```

Comment:
You might want to use the opportunity to add [`categories`](https://github.com/typst/packages/blob/main/CATEGORIES.md) and possibly also [`disciplines`](https://github.com/typst/packages/blob/main/DISCIPLINES.md).

Reply:
I just pushed a new typst.toml with the mentioned additions.